### DISCUSSION TITLE
[Prettier Plugin]  Is it possible to use with jinja2 and django templates?

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#11726] [Ideas]
The plugin depends on prettier. Prettier does not work with jinja2 and django templates.

Since we already have vscode tailwind extension and that does not depend on prettier and such, and it already supports identifying the tailwind clasess based on regex patterns.

I wonder if it's possible to have the vscode extension auto sort the tailwind classes.

**Comment by epicwhale** [comment]
Anyone has a solution to this? I am just not able to get the class auto sorting to work with a jinja2 template inside VS code, when I also have the jinja prettier plugin set up.

**Reply by msaints10** [reply]
My only somewhat tedious solution was to switch between the language of my vscode (html < - > django-html) and saving my changes once I finished making my changes. I hope they can solve it one day.
